.github/workflows/staging-smoke-tests.yaml

```diff
@@ -1,17 +1,67 @@
 name: DataRepo Staging Smoke Tests
 env:
-  K8_CLUSTER: 'staging-master'
-  # This must be defined for the bash redirection
-  GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
-  # This must be defined for the bash redirection
-  GOOGLE_SA_CERT: 'jade-dev-account.pem'
+  K8_CLUSTER: jade-master-us-central1
+  GOOGLE_APPLICATION_CREDENTIALS: /tmp/jade-dev-account.json
+  GOOGLE_SA_CERT: /tmp/jade-dev-account.pem
+  GOOGLE_CLOUD_PROJECT: terra-datarepo-staging
+  GOOGLE_CLOUD_DATA_PROJECT: terra-datarepo-staging-data
 on:
   workflow_dispatch: {}
 jobs:
   test-runner-staging:
     runs-on: ubuntu-latest
     steps:
-      - name: "Checkout jade-data-repo develop branch"
+      - name: "Checkout jade-data-repo"
         uses: actions/checkout@v2
+      - name: "Import Vault staging secrets"
+        uses: hashicorp/vault-action@v2.0.1
         with:
-          ref: develop
```

Comment:
Switch to latest tag before merge

Reply:
This actually needs to be done after this merge, as otherwise the required json files won't be available for the test runner.